src/App/Expression.cpp

```diff
@@ -2261,6 +2425,10 @@ Py::Object FunctionExpression::evaluate(const Expression *expr, int f, const std
         }
         unit = v1.getUnit();
         break;
+    case TRANSLATIONM:
+        if (v1.isDimensionlessOrUnit(Unit::Length) && v1.isDimensionlessOrUnit(Unit::Length) && v3.isDimensionlessOrUnit(Unit::Length))
```

Comment:
This should be:

```c++
if (v1.isDimensionlessOrUnit(Unit::Length) && v2.isDimensionlessOrUnit(Unit::Length) && v3.isDimensionlessOrUnit(Unit::Length))
```

Second element should be `v2` instead of `v1`.